package.json

```diff
@@ -44,15 +44,15 @@
   },
   "devDependencies": {
     "browserify": "16.5.0",
-    "grunt": "^1.0.0",
+    "grunt": "^1.0.4",
```

Comment:
I don't think it's a good idea using the '^'. We'll always use the edge version, and the behavior of the grunt lib might change.

Reply:
We use it with `^` in ImageSequencer. That is not a problem. It is advisable to always use the latest version.

Reply:
Correct, `^` won't update the major version so as @HarshKhandeparkar it is advisable to use it.